addons/website_event_questions/views/website_event_questions_templates.xml

```diff
@@ -6,15 +6,24 @@
         <!-- Attendee specific questions -->
         <xpath expr="//t[@name='attendee_loop']" position="inside">
             <t t-if="event.specific_question_ids">
-                <div class="row mb32">
-                    <div class="col-md-4" t-foreach='event.specific_question_ids' t-as='question'>
-                        <div class="input-group input-group-sm">
-                            <span class="input-group-addon"><t t-esc="question.title"/></span>
-                            <select t-attf-name="#{counter}-answer_ids-#{question.id}" class="form-control" required="true">
-                                <t t-foreach='question.answer_ids' t-as="answer">
-                                    <option t-esc='answer.name' t-att-value='answer.id'/>
-                                </t>
-                            </select>
+                <script>
+                    $(document).ready(function() {
```

Comment:
I used select2, because standard HTML select do not handle long text answer properly, option text goes outside the model(do you mean something else ?)